src/map/unit.c

```diff
@@ -823,28 +867,38 @@ static int unit_movepos(struct block_list *bl, short dst_x, short dst_y, int eas
 	return 1;
 }
 
-static int unit_setdir(struct block_list *bl, unsigned char dir)
+/**
+ * Sets the facing direction of a unit
+ * @param bl: unit to modify
+ * @param dir: the facing direction @see enum unit_dir
+ * @return 0: success, 1: failure
+ */
+static int unit_setdir(struct block_list *bl, enum unit_dir dir)
```

Comment:
please change the function signature as haru mentioned in his review.

Reply:
Done locally.